packages/client/src/lib/resources.ts

```diff
@@ -1,3 +1,7 @@
-export const HTTP_URL = process.env.REACT_APP_HTTP_URL;
+import { http } from 'utils/rest';
 
-export const getRooms = () => fetch(`${HTTP_URL}/rooms`).then((response) => response.json());
+export const getRooms = () => http.get('/rooms');
+
+export type DeleteRoomPlayload = { roomName: string; username: string };
```

Comment:
Does this need to be defined and exported?

Reply:
doen't need to be exported you are right

Reply:
I guess TS wants it to be defined though, right?

Reply:
I inlined the declaration yes, also mutate accept one single parameter (it needs to be object) otherwise it won't work, just to keep it in mind